sdk-docs/getting-started/running-the-iris-app-sdk.md

```diff
@@ -19,9 +19,17 @@ nx run [name-of-your-app]:serve
 ### 2. A browser window will open with this url <https://dev.manager.trackunit.com/iris-sdk-portal/main#runLocal>.
 You might need to login before it hits that page, use your developer credentials.
 
-### 3. The Trackunit Manager Developer Server _**Iris App Developer Portal**_ screen will appear.
+### 3. The screen for the developer-server called _**Iris App Developer Portal**_ will appear.
```

Comment:
I did like that the name Trackunit Manager was part of this title to make it explicit :D

Reply:
I agree, but at the same time the sentence was cluttered to read and I had to personally read it a few times over to understand what is part of a serve name and what was a noun :D